### ISSUE TITLE
Type Error possibly caused by latest published version

### ISSUE THREAD

**Issue by Ellosaur** [go-dima/serverless-plugin-conditional-functions#9]
Our last deployment that worked was `Jan 21st 2021`. Since then deployments all fail with the following errors. Noticed that v1.0.1 has just been published and our our package.json contains this: `"serverless-plugin-conditional-functions": "^1.0.0",` but no lock file
If we use a lock file and restore the use of version 1.0.0 normal service resumes

```
13:02:35    Type Error ---------------------------------------------
13:02:35   
13:02:35    TypeError: serverless.configSchemaHandler.defineFunctionProperties is not a function
13:02:35        at new serverlessPluginConditionalFunctions (/mnt/shared_workspace/proj/master/null/18/node_modules/serverless-plugin-conditional-functions/index.js:15:36)
13:02:35        at PluginManager.addPlugin (/mnt/shared_workspace/proj/master/null/18/node_modules/serverless/lib/classes/PluginManager.js:94:28)
```

```
13:02:35    Your Environment Information ---------------------------
13:02:35       Operating System:          linux
13:02:35       Node Version:              14.15.4
13:02:35       Framework Version:         1.83.2 (local)
13:02:35       Plugin Version:            3.8.4
13:02:35       SDK Version:               2.3.2
13:02:35       Components Version:        2.34.9
```

**Comment by dsciacca** [comment]
Our deployments are also failing since the recent deployment but with a different error:

```
Serverless Error -----------------------------------------

Serverless plugin "serverless-plugin-conditional-functions" initialization errored: Cannot read property 'defineFunctionProperties' of undefined
```
```
Your Environment Information ---------------------------
Operating System:          linux
Node Version:              10.1.0
Serverless Version:        1.48.3
Enterprise Plugin Version: 1.3.11
Platform SDK Version:      2.3.2
```

I can open a separate issue if necessary, please let me know. Thanks!

**Comment by go-dima** [comment]
@Ellosaur, @dsciacca - Thanks for your feedback!

v1.0.1 (and v1.0.2) include schema validation support.

As far as I can tell, **defineFunctionProperties** was added in serverless v2.10.0: [Release here](https://github.com/serverless/serverless/releases/tag/v2.10.0 ).

So locking to v1.0.0 seems to be the correct way to go.

**Comment by go-dima** [comment]
I published a new version: **v1.0.4** (#11) which supports all serverless versions as far as I can tell, so you can use latest again.

Fell free to contact me if any issue arises!
